cypress/integration/page.spec.js

```diff
@@ -1,7 +1,81 @@
-// WRITE TESTS HERE
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+const buttons = {
+  clear: 'Clear',
+  remove: 'Remove',
+  select: 'Select'
+};
+
+Cypress.Commands.add('selectItemBtn', (btn) => {
+  cy.contains('li', goodsFromServer[8])
+    .contains(btn);
+});
 
 describe('Page', () => {
-  it('should be visitable', () => {
+  beforeEach(() => {
     cy.visit('/');
   });
+
+  it('should have only original items on the list', () => {
+    cy.get('li')
+      .should('have.length', goodsFromServer.length);
+  });
+
+  it('should have selected Jam item by default', () => {
+    cy.contains('h1', `${goodsFromServer[8]} is selected`)
+      .should('exist');
```

Comment:
```suggestion
    cy.get('h1')
      .should.contain(`${goodsFromServer[8]} is selected`);
```

Може не так записується, але виглядає логічніше

Reply:
так, дійсно краще. там більше немає інших h1